### DISCUSSION TITLE
pretixSCAN check-in without selecting an event

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3590] [Q&A]
Hello!
I have configured pretix for using 7 or more events per day and I want to use pretixSCAN for the check-in process.
I noticed that in pretixSCAN I have to select the event first and then scan the ticket which is not very user friendly in my case with 7 or more events per day. Can I scan the tickets without firstly selecting an event ?

Please consider that the option "Automatic event selection" in PretixScan->Settings is greyed out and I cannot enable it (if this could solve my issue).
[screenshot](https://prnt.sc/BeHmVqDUamdC)

**Comment by raphaelm** [comment]
> Can I scan the tickets without firstly selecting an event ?

No, an event always needs to be selected first.

**Reply by alemao8** [reply]
ok I see. Can I enable the automatic event selection option to check if this solves my issue ?
Finally, there is an option of enabling Advanced mode (for selecting multi events). This functionality makes a list of the events but I cannot add more than 5 events. Is this correct ? Can I extend this limitation ?

**Reply by alemao8** [reply]
I have enabled the automatic event selection. No issue with that.
Also there is no limitation. I had to select an event with a different slug.
